homeassistant/components/roomba/irobot_base.py

```diff
@@ -72,6 +74,10 @@ def __init__(self, roomba, blid):
         self._version = self.vacuum_state.get("softwareVer")
         self._sku = self.vacuum_state.get("sku")
 
+        self._run_stats = self.vacuum_state.get("bbrun")
+        self._mission_stats = self.vacuum_state.get("bbmssn")
+        self._battery_stats = self.vacuum_state.get("bbchg3")
```

Comment:
Should these statistics only be fetched on initialisation, and not updated after?

Reply:
No they should still update after initialisation. I will have to find out how to do that.